modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-api/src/main/java/com/liferay/portal/search/tuning/rankings/util/RankingUtil.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.portal.search.tuning.rankings.web.internal.util;
+package com.liferay.portal.search.tuning.rankings.util;
```

Comment:
i think we typically try to stay away from having a Util class in an API module... most classes in our APIs are interfaces and contain very little actual logic. maybe this can be extracted to api/impl... see if you can find another example/pattern in some other modules on how they deal with Utils